pandas/tests/io/json/test_pandas.py

```diff
@@ -49,11 +39,18 @@ def assert_json_roundtrip_equal(result, expected, orient):
 )
 @pytest.mark.filterwarnings("ignore:the 'numpy' keyword is deprecated:FutureWarning")
 class TestPandasContainer:
-    @pytest.fixture(autouse=True)
-    def setup(self):
-        self.categorical = _cat_frame.copy()
+    @pytest.fixture
+    def categorical_frame(self):
+        _seriesd = tm.getSeriesData()
+
+        _frame = DataFrame(_seriesd)
 
-        yield
+        _cat_frame = _frame.copy()
```

Comment:
this copy is probably unnecessary too.  ill remove it in my next Assorted Cleanups branch

Reply:
Thanks 🙏